### DISCUSSION TITLE
Clarifying Ingress NGINX configuration

### DISCUSSION THREAD

**Discussion by cer** [linkerd/linkerd2#10949] [Q&A]
The [documentation](https://linkerd.io/2.13/tasks/using-ingress/#nginx) seems a little contradictory:

> Nginx can be meshed normally, but....

What precisely does 'meshed normally' mean?

Given that this is then followed by:

> but the [nginx.ingress.kubernetes.io/service-upstream](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#service-upstream) annotation should be set to "true".

And 

> If using [this Helm chart](https://artifacthub.io/packages/helm/ingress-nginx/ingress-nginx), note the following.

I think this could be clearer and less contradictory.

**Comment by wmorgan** [comment]
Meshed "normally" is meant to be in contrast to meshed in ingress mode... this may have been clearer earlier on before the introductory paragraph was moved so far up (this doc has evolved quite a bit). 

Thanks for the note. We'll clean this up.

**Reply by cer** [reply]
Thanks. It would be clearer if this section was written as 'switch/case statement' with the normal mode (linked to docs) being the last, default case

**Comment by wmorgan** [comment]
I've updated the docs to hopefully be clearer.
